scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala

```diff
@@ -89,9 +89,21 @@ object Cli {
     //   (or if the `version` is not specified).
     val runner: ScalafmtRunner = options.version match {
       case None => ScalafmtCoreRunner
-      case Some(v) if v == Versions.version =>
-        ScalafmtCoreRunner
-      case _ => ScalafmtDynamicRunner
+      case Some(v) =>
+        if (v == Versions.version) ScalafmtCoreRunner
+        else {
+          val isNativeImage =
+            "true" == System.getProperty("scalafmt.native-image", "false")
+          if (isNativeImage) {
```

Comment:
I like it 👍

Reply:
This doesn't seem to work yet, I'll try to fix it before merging.

Reply:
OK, should be fixed now!
<img width="913" alt="Screenshot 2019-11-27 at 14 51 02" src="https://user-images.githubusercontent.com/1408093/69733525-5c67f380-1125-11ea-9f39-2f5e85ea3162.png">